pkg/executables/govc_test.go

```diff
@@ -196,11 +202,25 @@ func (dt *deployTemplateTest) assertDeployTemplateError(t *testing.T) {
 	}
 }
 
+func (dt *deployTemplateTest) assertDeployOptsMatches(t *testing.T) {
+	expected, err := executables.GetDeployOptions(dt.network)
```

Comment:
this is really a super valid code, since it's using the code we are suppose to be testing to generate the expected output

I would suggest just hardcoding the json body here in a string/[]byte (even the network value) and comparing the file against that